ui/src/getPackageInfo.ts

```diff
@@ -55,12 +56,15 @@ export function getPackageInfo(ctx?: ExtensionContext): { extensionName: string,
 
     const extensionId: string = `${packageJson.publisher}.${packageJson.name}`;
 
-    return { extensionName, extensionVersion, aiKey, extensionId };
+    return { extensionName, extensionVersion, aiKey, extensionId, bugsUrl };
 }
 
 interface IPackageJson {
     version?: string;
     name?: string;
     publisher?: string;
     aiKey?: string;
+    bugs?: {
```

Comment:
[Docs](https://docs.npmjs.com/files/package.json#bugs) say the following about bugs:
> If you want to provide only a url, you can specify the value for “bugs” as a simple string instead of an object.

I don't _think_ any of our repos specify "bugs" as a string, but probably worth being safe